samples/java-spring-eventsourced-customer-registry-subscriber/docker-compose.yml

```diff
@@ -4,12 +4,13 @@ services:
   # on the same local machine as the java-spring-customer-registry
   kalix-proxy:
     image: gcr.io/kalix-public/kalix-proxy:1.1.8
-    command: -Dconfig.resource=dev-mode.conf -Dlogback.configurationFile=logback-dev-mode.xml #-Dkalix.proxy.eventing.support=google-pubsub-emulator
```

Comment:
removed `kalix.proxy.eventing.support=google-pubsub-emulator`, not needed in this service